src/views/ModelPlan/Status/index.tsx

```diff
@@ -42,6 +42,8 @@ const Status = () => {
     status: Yup.string().required('Enter a role for this team member')
   });
 
+  const readOnly = history.location.hash === '#read-only';
```

Comment:
This can use the `useLocation()` hook instead of the `useHistory()` hook.  Just a little more concise with the former.